features/FEATURE_BLE/targets/TARGET_STM/TARGET_NUCLEO_WB55RG/stm32wb_HCIDriver.cpp

```diff
@@ -0,0 +1,902 @@
+                /* initialize ACL buffer accounting */
+                hciCoreCb.availBufs = hciCoreCb.numBufs;
+                // read the states and state combinations supported by the link
+                // layer of the controller
+                HciLeReadSupStatesCmd();
+                break;
+
+            case HCI_OPCODE_LE_READ_SUP_STATES:
+                // store supported state and combination in the runtime parameters
+                // of the stack
+                memcpy(hciCoreCb.leStates, pMsg, HCI_LE_STATES_LEN);
+                // read the total of whitelist entries that can be stored in the
+                // controller.
+                HciLeReadWhiteListSizeCmd();
+                break;
+
+            case HCI_OPCODE_LE_READ_WHITE_LIST_SIZE:
+                // store the number of whitelist entries in the stack runtime
+                // parameters
+                BSTREAM_TO_UINT8(hciCoreCb.whiteListSize, pMsg);
+
+                // Read the LE features supported by the controller
+                HciLeReadLocalSupFeatCmd();
+                break;
+
+            case HCI_OPCODE_LE_READ_LOCAL_SUP_FEAT:
+                // Store the set of LE features supported by the controller
+                BSTREAM_TO_UINT16(hciCoreCb.leSupFeat, pMsg);
+                // read the total number of address translation entries which can be
+                // stored in the controller resolving list.
+                hciCoreReadResolvingListSize();
+                break;
+
+            case HCI_OPCODE_LE_READ_RES_LIST_SIZE:
+                // store the number of address translation entries in the stack
+                // runtime parameter
+                BSTREAM_TO_UINT8(hciCoreCb.resListSize, pMsg);
+
+                // read the Controller's maximum supported payload octets and packet
+                // duration times for transmission and reception
+                hciCoreReadMaxDataLen();
+                break;
+
+            case HCI_OPCODE_LE_READ_MAX_DATA_LEN:
+              {
+                // store payload definition in the runtime stack parameters.
+                uint16_t maxTxOctets;
+                uint16_t maxTxTime;
+
+                BSTREAM_TO_UINT16(maxTxOctets, pMsg);
+                BSTREAM_TO_UINT16(maxTxTime, pMsg);
+
+                /* use Controller's maximum supported payload octets and packet duration times
+                * for transmission as Host's suggested values for maximum transmission number
+                * of payload octets and maximum packet transmission time for new connections.
+                */
+                HciLeWriteDefDataLen(maxTxOctets, maxTxTime);
+              }
+              break;
+
+            case HCI_OPCODE_LE_WRITE_DEF_DATA_LEN:
+                if (hciCoreCb.extResetSeq)
+                {
+                    /* send first extended command */
+                    (*hciCoreCb.extResetSeq)(pMsg, opcode);
```

Comment:
the extResetSeq needs a command to start it off, please have a look at:
https://github.com/paul-szczepanek-arm/mbed-os/commit/b96c0ac986234457c0c2450ce79a3c9bd1336290

Reply:
Ok - thanks will do